backend/gameplay/models.py

```
@@ -166,5 +161,40 @@ class UserGameplaySettings(models.Model):
+		
+	@classmethod
+	@database_sync_to_async
+	def async_create_timer(cls, game_room_id, is_timer_running = True):
+		GameplayTimerTask.objects.create(game_room_id=game_room_id, is_running=is_timer_running)
+	
+	def is_timer_running(self):
+		return self.is_running
+	
+	@database_sync_to_async
+	def async_start(self):
+		setattr(self, "is_running", True)
```

Ruff (0.8.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)